scheduledJobs/giveApples.js

```diff
@@ -0,0 +1,332 @@
+const cron = require('node-cron');
+const Promise = require('promise');
+const { batchItems, fetchAllMessagesByChannelSince, getGuildIdsWithoutBAE, sendGiveApplesEmbed } = require('../common/discordutil');
+const { DateTime } = require('luxon');
+const { MessageAttachment } = require('discord.js');
+const appleConfig = require('./giveApplesConfig.json');
+
+const ALL_GUILD_IDS = getGuildIdsWithoutBAE();
+
+let CURRENT_APPLE_TASK = null;
+
+const getLogbookMessagesToCheck = async (messages) => {
+    const ignoreReactionName = 'yoongerine';
```

Comment:
can all code in the world have some BTS reference like this? 
![image](https://user-images.githubusercontent.com/16905302/200115952-80c4898c-dd22-4776-9a08-dcc0a1776546.png)